include/fmt/format.h

```diff
@@ -1850,6 +1852,17 @@ auto write(OutputIt out, const T& value) -> typename std::enable_if<
   return formatter<T>().format(value, ctx);
 }
 
+template <typename Char, typename OutputIt, typename T,
+          FMT_ENABLE_IF(has_fallback_formatter<
+                        T, basic_format_context<OutputIt, Char>>::value)>
+auto write(OutputIt out, const T& value) -> typename std::enable_if<
+    mapped_type_constant<T, basic_format_context<OutputIt, Char>>::value ==
+        type::custom_type,
+    OutputIt>::type {
+  basic_format_context<OutputIt, Char> ctx(out, {}, {});
+  return fallback_formatter<T>().format(value, ctx);
+}
```

Comment:
This overload can be merged into the one above replacing `formatter<T>` with

```c++
conditional_t<has_formatter<T, Context>::value,
                         typename Context::template formatter_type<T>,
                         fallback_formatter<T, char_type>>
```

Reply:
done